README.md

```diff
@@ -4,6 +4,7 @@ This repository serves as a check-in for attendees of the **Git Workshop for Mod
 
 ---
 ## Attendees
+- [Jairus Carle Isaguirre](https://github.com/RandomGuyJCI)

```

Comment:
Actually, lemme rescind the previous approval. Let's use asterisks here for consistency with the other PRs.
```suggestion
* [Jairus Carle Isaguirre](https://github.com/RandomGuyJCI)
```